community/kernel/src/test/java/org/neo4j/kernel/impl/api/index/IndexPopulationJobTest.java

```diff
@@ -700,9 +695,17 @@ private DoubleLongRegister indexUpdatesAndSize( Label label, String propertyKey
         {
             int labelId = statement.readOperations().labelGetForName( label.name() );
             int propertyKeyId = statement.readOperations().propertyKeyGetForName( propertyKey );
-            DoubleLongRegister result =
-                    statement.readOperations().indexUpdatesAndSize( IndexDescriptorFactory.of( labelId, propertyKeyId ),
-                            Registers.newDoubleLongRegister() );
+            DoubleLongRegister result = newDoubleLongRegister(0, 0);
+            try
+            {
+                result = statement.readOperations()
+                        .indexUpdatesAndSize( IndexDescriptorFactory.of( labelId, propertyKeyId ),
+                                newDoubleLongRegister() );
+            }
+            catch (IndexNotFoundKernelException e)
+            {
+                // Tests ignore this
```

Comment:
Oops, that comment was apparently a bit to fast. The return value will be "zero" if no index exist. This hides the expected behaviour (introduced in this PR), namely seeing the `IndexNotFoundKernelException`. Let's adapt this test to this behaviour change.